src/main/scala/cognite/spark/v1/FlexibleDataModelRelationUtils.scala

```diff
@@ -531,18 +548,14 @@ object FlexibleDataModelRelationUtils {
               propertyName,
               corePropDef,
               instanceSpace)
-          case t if t.isList => toInstantPropertyValueOfList(row, schema, propertyName, corePropDef)
-          case _ => toInstantPropertyValueOfNonList(row, schema, propertyName, corePropDef)
+          case t if t.isList => toInstancePropertyValueOfList(row, schema, propertyName, corePropDef)
+          case _ => toInstancePropertyValueOfNonList(row, schema, propertyName, corePropDef)
         }
       case _: PropertyDefinition.ConnectionDefinition =>
-        val fieldIndex = Try(schema.fieldIndex(propertyName))
-        val nullAtIndex = fieldIndex.map(row.isNullAt).getOrElse(true)
-        if (nullAtIndex) {
-          Right(None)
-        } else {
+        lookupFieldInRow(row, schema, propertyName, true) { _ =>
           extractDirectRelation(propertyName, "Connection Reference", schema, instanceSpace, row)
```

Comment:
not sure how to read syntax here :thinking: 
`lookupFieldInRow`'s result is ignored? (looking at `_ =>`)

Reply:
Yes, the index that `lookupFieldInRow` provides to the continuation function is ignored. This is the rust convention that I put in unthinkingly. Is there a scala convention for patterns that ignore the argument?

Reply:
I think that's the convention, but was unsure if it should be ignored

Reply:
It's ignored because `extractDirectRelation` does its own index lookup, by the way